services/movie_session.py

```diff
@@ -42,3 +42,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    tickets = (MovieSession.objects.get(
+        id=movie_session_id
+    ).ticket_set.values("row", "seat"))
+    return list(tickets)
```

Comment:
Consider handling the case where the MovieSession with the given movie_session_id does not exist. Using get() without handling exceptions can lead to a DoesNotExist exception if the object is not found.